### PR TITLE
cleanup drafts when user logs out

### PR DESCRIPTION
**Pull request by connyduck**

### CODE REVIEW

File: app/src/main/java/com/keylesspalace/tusky/components/drafts/DraftHelper.kt [hunk 1]
```diff
@@ -126,6 +127,15 @@ class DraftHelper @Inject constructor(
                 .andThen(draftDao.delete(draft.id))
     }
 
+    fun deleteAllDraftsAndAttachmentsForAccount(accountId: Long) {
+        draftDao.loadDraftsSingle(accountId)
```
Comment (charlag) [inline app/src/main/java/com/keylesspalace/tusky/components/drafts/DraftHelper.kt:131]: I would probably rather write SQL for this but I don't think it matters much

**Review by charlag — APPROVED** [review]

File: app/src/main/java/com/keylesspalace/tusky/components/drafts/DraftHelper.kt (hunk 1 above)
Comment (connyduck) [reply app/src/main/java/com/keylesspalace/tusky/components/drafts/DraftHelper.kt:131]: I'm not sure I understand what you mean? The SQL statement is defined in DraftDao?

Comment (charlag) [reply app/src/main/java/com/keylesspalace/tusky/components/drafts/DraftHelper.kt:131]: I mean for the whole deletion thing, sorry for being unclear

Comment (connyduck) [reply app/src/main/java/com/keylesspalace/tusky/components/drafts/DraftHelper.kt:131]: Ah! But I need to query the drafts so I can delete the attachment files before deleting the drafts in the db. I can't query & delete in 1 statement, can I?

Comment (charlag) [reply app/src/main/java/com/keylesspalace/tusky/components/drafts/DraftHelper.kt:131]: I think you can get all those attachments but I think it's a rather complicated query

Comment (connyduck) [reply app/src/main/java/com/keylesspalace/tusky/components/drafts/DraftHelper.kt:131]: I rather leave it like this then

**Pull request merged**